sdk/keyvault/azure-security-keyvault-certificates/CMakeLists.txt

```diff
@@ -40,9 +40,11 @@ if(FETCH_SOURCE_DEPS)
         add_subdirectory(${oneFolder} EXCLUDE_FROM_ALL)
     endforeach()
 elseif(NOT AZ_ALL_LIBRARIES)
-  find_package(azure-core-cpp "1.2.0" CONFIG QUIET)
+  find_package(azure-core-cpp "1.5.0" CONFIG QUIET)
+  find_package(azure-identity-cpp "1.2.0" CONFIG QUIET)
   if(NOT azure-core-cpp_FOUND)

```

Comment:
Why the new dependency on identity? Was the dependency already there but hidden? Why didn't the missing dependency make a difference?